core/tests/webdriverio_utils/ClassroomPage.js

```diff
@@ -40,6 +42,10 @@ var ClassroomPage = function() {
       expect(topicSummaryTiles.length).toEqual(0);
     }
   };
+
+  this.takeDiagnosticTest = async function() {
```

Comment:
This is misnamed, it suggests that the learner has completed the whole test. I suggest calling it "launchDiagnosticTestPage()" or similar.

Reply:
Updated